src/ProjectNeon/Assets/Features/Effects/Events/AttackPerformed.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+public sealed class AttackPerformed
+{
+    public Attack Attack { get; }
```

Comment:
No behavior allowed in messages. Only data.

Reply:
@SilasReinagel How am I supposed to read the value of `Attack` then?